go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/red-hat-storage/ocs-client-operator
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.4
```

Comment:
What is the reason we need to add toolchain to our go.mod file?

Reply:
The toolchain directive gets automatically added upon running go commands (e.g., `go mod tidy`) for versions starting in 1.21.

Reply:
ok, thanks for the explanation